tools/sync.js

```diff
@@ -0,0 +1,214 @@
+/*
+Name: Ethereum Blockchain syncer
+Version: .0.0.1
+This file will start syncing the blockchain from the node address you provide in the conf.json file.
+Please read the README in the root directory that explains the parameters of this code
+*/
+require( '../db.js' );
+var etherUnits = require("../lib/etherUnits.js");
+var BigNumber = require('bignumber.js');
+
+var fs = require('fs');
+var Web3 = require('web3');
+
+var mongoose        = require( 'mongoose' );
+var Block           = mongoose.model( 'Block' );
+var Transaction     = mongoose.model( 'Transaction' );
+
+//Just listen for latest blocks and sync from the start of the app.
+var listenBlocks = function(config) {
+    var newBlocks = web3.eth.filter("latest");
+    newBlocks.watch(function (error, log) {
+    if(error) {
+        console.log('Error: ' + error);
+    } else if (log == null) {
+        console.log('Warning: null block hash');
+    } else {
+      console.log('Found new block: ' + log);
+      grabBlock(config,web3,log);
+      updatedEndBlock(config,log);
+    }
+  });
+}
+//Grab latest block info and it transactions and write to db
+var grabBlock = function(config, web3, blockHashOrNumber) {
```

Comment:
`grabBlock` -> `readBlock` maybe?

Reply:
piped the whole function into one call no need for variable now.